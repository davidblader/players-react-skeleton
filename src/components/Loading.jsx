import React from 'react';
import Image from './Image';
import AnimalCrossingContainer from './AnimalCrossingContainer';
import AnimalCrossingHeader from './AnimalCrossingHeader';

const Loading = () => (
  <AnimalCrossingContainer>
    <AnimalCrossingHeader>Loading...</AnimalCrossingHeader>
    <Image src="gyroid" />
  </AnimalCrossingContainer>
);

export default Loading;
