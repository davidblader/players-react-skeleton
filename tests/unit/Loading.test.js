import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../../src/components/Loading';

test('Loading shallow renders', () => {
  const component = shallow(<Loading />);
  expect(component).toMatchSnapshot();
});
