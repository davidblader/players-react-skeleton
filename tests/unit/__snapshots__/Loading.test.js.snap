// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading shallow renders 1`] = `
<AnimalCrossingContainer>
  <AnimalCrossingHeader>
    Loading...
  </AnimalCrossingHeader>
  <Image
    src="gyroid"
  />
</AnimalCrossingContainer>
`;
